actors/market/src/lib.rs

```diff
@@ -1153,6 +1157,52 @@ impl Actor {
         let found = rt.state::<State>()?.get_proposal(rt.store(), params.id)?;
         Ok(GetDealVerifiedReturn { verified: found.verified_deal })
     }
+
+    /// Fetches activation state for a deal.
+    /// This will be available from when the proposal is published until an undefined period after
+    /// the deal finishes (either normally or by termination).
+    /// Outside that period, returns USR_NOT_FOUND.
```

Comment:
This comment needs updating to reflect that it will return `EX_DEAL_EXPIRED` after the period.